clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoAdminClient.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.client;
+
+import com.datastrato.gravitino.MetalakeChange;
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.SupportsMetalakes;
+import com.datastrato.gravitino.dto.requests.MetalakeCreateRequest;
+import com.datastrato.gravitino.dto.requests.MetalakeUpdateRequest;
+import com.datastrato.gravitino.dto.requests.MetalakeUpdatesRequest;
+import com.datastrato.gravitino.dto.responses.DropResponse;
+import com.datastrato.gravitino.dto.responses.MetalakeListResponse;
+import com.datastrato.gravitino.dto.responses.MetalakeResponse;
+import com.datastrato.gravitino.exceptions.MetalakeAlreadyExistsException;
+import com.datastrato.gravitino.exceptions.NoSuchMetalakeException;
+import com.google.common.base.Preconditions;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Gravitino Client for the administrator to interact with the Gravitino API, allowing the client to
+ * list, load, create, and alter Metalakes.
+ *
+ * <p>Normal users should use {@link GravitinoClient} to connect with the Gravitino server.
+ */
+public class GravitinoAdminClient extends GravitinoClientBase implements SupportsMetalakes {
```

Comment:
Shall we also support load metalake for admin client?

Reply:
Do you mean the "loadMetalake()" method? It is a basis function I think, which is to load the metalake information from Gravitino server side. Without it, many functions are broken. Besides, the admin client has the full capability for metalake, including list, load, create, alter, drop, etc. So I think it needs be there.

Reply:
Sorry about. I saw the load method is already implemented in the base class.